### PR TITLE
Feature/eui 444 login filter

### PR DESCRIPTION
**Pull request by elviani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/cases/containers/case-list/case-list.component.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Store, select } from '@ngrx/store';
 import { Observable, Subscription, combineLatest } from 'rxjs';
 import { Jurisdiction, CaseType, CaseState, SearchResultView, PaginationMetadata } from '@hmcts/ccd-case-ui-toolkit';
 import { FormGroup } from '@angular/forms';
+import { DefinitionsService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services/definitions/definitions.service';
```
Comment (praveenrajendran101) [inline src/cases/containers/case-list/case-list.component.ts]: Directly referenced from CCD file. It should be @hmcts/ccd-case-ui-toolkit.

File: src/cases/containers/case-list/case-list.component.ts [hunk 2]
```diff
@@ -148,25 +155,26 @@ export class CaseListComponent implements OnInit, OnDestroy {
...
-          caseType: caseTypeGroupFromLS,
-          metadataFields: metadataFieldsGroupFromLS,
-          formGroup: {
-            value: formGroupFromLS
-          },
-          page: this.page,
-          view: 'WORKBASKET'
-        }
-      };
+    if (this.selected) {
```
Comment (praveenrajendran101) [inline src/cases/containers/case-list/case-list.component.ts]: Local storage is needed for remembering previous selections. Please refer my latest fix that is on master.

File: api/auth/index.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export function doLogout(req, res, status = 302) {
     res.clearCookie(cookieToken)
     res.clearCookie(cookieUserId)
     req.session.user = null
+    delete req.session.auth
```
Comment (praveenrajendran101) [inline api/auth/index.ts]: Just trying to understand the need for auth deletion

File: src/cases/containers/case-list/case-list.component.ts (hunk 1 above)
Comment (elviani) [reply src/cases/containers/case-list/case-list.component.ts]: Andy have approved this changes for now, until the ccd team has exported the service.

**Pull request merged**